### DISCUSSION TITLE
rename examples/main to llama-cpp ?

### DISCUSSION THREAD

**Discussion by juhp** [ggml-org/llama.cpp#7956] [Ideas]
See https://bugzilla.redhat.com/show_bug.cgi?id=2255828#c12
and https://src.fedoraproject.org/rpms/llama-cpp/blob/rawhide/f/llama-cpp.spec

It would be nice to have the main executable included in the default build and perhaps some of the other more prominent examples too.

**Comment by ggerganov** [comment]
It was renamed recently to `llama-cli`
